docs/superpowers/plans/2026-08-14-omarchy-quattro-compat.md

```
@@ -0,0 +1,685 @@
+(On plain Lua setups without Omarchy's helpers, register the command
+on the start hook:
+`hl.on("hyprland.start", function() hl.exec_cmd([[nwg-notifications --persist]]) end)`.)
+
+**Migrating to Omarchy 4.0:** the Quattro migration generates the new
+`.lua` config files but does **not** carry custom `exec-once` lines
+across from `autostart.conf`. For nwg-notifications the loss is partly
+masked — the D-Bus service files still auto-activate the daemon on the
+first notification once the name is free — but activation only wins if
+nothing else owns `org.freedesktop.Notifications` (see the Omarchy 4.0
+section below), so re-add the autostart line explicitly.
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)